<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HDF5BrowserDialog</class>
 <widget class="QDialog" name="HDF5BrowserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigateur HDF5</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="headerLabels" stdset="0">
      <stringlist>
       <string>Nom</string>
       <string>Type</string>
       <string>Chemin</string>
      </stringlist>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutCube">
     <item>
      <widget class="QLabel" name="labelCube">
       <property name="text">
        <string>Cube path :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_cube">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_select_cube">
       <property name="text">
        <string>→ Cube</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutWl">
     <item>
      <widget class="QLabel" name="labelWl">
       <property name="text">
        <string>Wavelength path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_wl">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_select_wl">
       <property name="text">
        <string>→ Wavelength</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMeta">
     <item>
      <widget class="QLabel" name="labelMeta">
       <property name="text">
        <string>Metadata Path :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_meta">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_select_meta">
       <property name="text">
        <string>→ Metadata</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QLabel" name="label_wl_dim">
       <property name="text">
        <string>wl dim : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_channel_wl">
       <item>
        <property name="text">
         <string>Last</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>First</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(194, 194, 194);</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
